---a/net/minecraft/world/IWorldNameable.java
+++b/net/minecraft/world/IWorldNameable.java
@@ -2,12 +2,12 @@
 
 import net.minecraft.util.IChatComponent;
 
-public interface IWorldNameable
-{
+public interface IWorldNameable {
+
     String getName();
 
     boolean hasCustomName();
 
     IChatComponent getDisplayName();
-}
+}
